frontend/src/components/RegistrationForm/RegistrationForm.js

```diff
@@ -0,0 +1,235 @@
+                    InputProps={{
+                      endAdornment: (
+                          <InputAdornment position="end">
+                            <IconButton
+                              onClick={handleClickShowPassword}
+                              onMouseDown={handleMouseDownPassword}
+                            >
+                              {
+                                showPassword ? 
+                                  <img src={visibilityOffIcon} alt="Password visibility icon" width={24} height={24} />
+                                  : <img src={visibilityIcon} alt="Password visibility off icon" width={24} height={24} />
+                              }
+                            </IconButton>
+                        </InputAdornment>
+                      )
+                    }}
+                  />
+                </Grid>
+                <Grid item xs={12}>
+                  <TextField
+                    className="form__last-text-field"
+                    required
+                    error={showError.errorConfirmPassword}
+                    helperText={showError.errorConfirmPassword ? "Passwords do not match" : ""}
+                    type={showConfirmPassword ? "text" : "password"}
+                    id="confirm-password-field"
+                    label="Confirm Password"
+                    placeholder="**********"
+                    variant="outlined"
+                    fullWidth
+                    onChange={handleConfirmPasswordChange}
+                    InputProps={{
+                      endAdornment: (
+                          <InputAdornment position="end">
+                            <IconButton
+                              onClick={handleClickShowConfirmPassword}
+                              onMouseDown={handleMouseDownPassword}
+                            >
+                              {
+                                showConfirmPassword ? 
+                                  <img src={visibilityOffIcon} alt="Password visibility icon" width={24} height={24} />
+                                  : <img src={visibilityIcon} alt="Password visibility off icon" width={24} height={24} />
+                              }
+                            </IconButton>
+                        </InputAdornment>
+                      )
+                    }}
+                  />
+                </Grid>
+              </Grid>
+            </Grid>
+            <Grid item xs={12}>
+              <Grid container spacing={2}>
+                <Grid item xs={6}>
+                  <Button
+                    type="submit"
+                    variant="contained"
+                    color="primary"
+                    fullWidth
+                  >
+                    Register
+                  </Button>
+                </Grid>
+                <Grid item xs={6}>
+                  <Button variant="outlined" fullWidth href="/">
+                    Login
```

Comment:
wouldn't 'Back to Login' be more appropriate here?

Reply:
Ok, chief